src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/TabularLinearFunctions/Builder.hpp

```diff
@@ -0,0 +1,29 @@
+template< typename ParentBuilder >
+class Builder: 
+  public Tabulated< Data >::Builder< Builder< ParentBuilder > >,
```

Comment:
As before: protected inheritance.

Reply:
I can't do that; the compiler complains with `cannot cast protected base class`.